### ISSUE TITLE
retrieve and rank createSolrClient does not pass authentication correctly.

### ISSUE THREAD

**Issue by jwickens** [watson-developer-cloud/node-sdk#321]
I am receiving a 401 unauthorized when using createSolrClient:

Assume `var retrieve = new watson.RetrieveAndRankV1();` with the credentials through bluemix.

`retreive.listClusters(cb)` works 

but 

```javascript
var solrClient = retrieve.createSolrClient(cluster_params);
solrClient.get('fcselect', query_params_plus_rankerId, cb);
```

Does not.

Looking at the source, the method is looking for this._options.token, perhaps this no longer exists?

**Comment by nfriedly** [comment]
@kognate this is my fault, I'll have a fix out shortly

**Comment by nfriedly** [comment]
@jwickens Thanks for the report and sorry for the bug. It should be fixed in `v2.4.6` once the CI loop finishes.

**Comment by jwickens** [comment]
@nfriendly all good now, thanks!
